.travis.yml

```diff
@@ -1,8 +1,16 @@
 language: php
+
 php:
+  - 5.6
   - 5.5
   - 5.4
   - hhvm
+
+matrix:
+  fast_finish: true
+  allow_failures:
+    - php: hhvm
```

Comment:
This can still merge cleanly but I have hhvm working so would rather not include it in the allow failures. Can you merge in master and to see if this still passes on Travis?